src/App.vue

```diff
@@ -347,6 +347,33 @@ export default Vue.extend({
             if(event.key.toLowerCase() == "contextmenu"){
                 this.appStore.isContextMenuKeyboardShortcutUsed  = true;
             }
+
+            // Handling the notification for doing duplication with drag and drop.
+            // We don't really care if another key is hit along ctrl/meta, we only look that
+            // we are currently in a drag and drop action, and notify the current caret candidate for drop that
+            // the action requires frame duplication.
+            if(this.appStore.isDraggingFrame && (event.ctrlKey || event.metaKey)){
+                addDuplicateActionOnFramesDnD();
+                event.preventDefault();
+                event.stopImmediatePropagation();
+                event.stopPropagation();
+                return;
+            }
+        });
+
+        // There are only a few cases when we need to handle key up events
+        window.addEventListener("keyup", (event) => {
+            // Handling the notification for not doing duplication anymore with drag and drop.
+            // We don't really care if another key is hit along ctrl/meta, we only look that
```

Comment:
Also need to adjust the comment here (and above) to say Option or Alt instead of Meta.

Reply:
I've added them on the condition,  I had forgotten them. Note that "ctrlKey" an alike can't be used for "keyup" events, because if the modifier key is the only one pressed, the modifier flag on the event will be false. We need to use the key code/value instead.